packages/dotnet-sdk/src/TeamsFx/MsGraphAuthProvider.cs

```diff
@@ -5,14 +5,17 @@ namespace Microsoft.TeamsFx;
 using global::Azure.Core;
 
 using Microsoft.Extensions.Logging;
-using Microsoft.Graph;
+using Microsoft.Kiota.Abstractions;
```

Comment:
Is it necessary to explicitly reference `Microsoft.Kiota*` packages in csproj to avoid potential version conflict?

Reply:
Yes, since `IAuthenticationProvider is no longer defined in `Microsoft.Graph`, but defined in `Microsoft.Kiota` instead

Reply:
Ignore me. Dotnet's package manager should already handle this.